tools/Makefile

```diff
@@ -98,8 +97,11 @@ $(curdir)/genext2fs/compile := $(curdir)/libtool/compile
 $(curdir)/gengetopt/compile := $(curdir)/libtool/compile
 $(curdir)/gmp/compile := $(curdir)/libtool/compile
 $(curdir)/isl/compile := $(curdir)/gmp/compile
+<<<<<<< HEAD
```

Comment:
here

Reply:
yeah, no idea how that snook in ... 0bviously during rebase :p